### PR TITLE
add comprehensive error message

### PR DESCRIPTION
**Pull request by daiki44**
## What?
SSIA
WAK-139

## TODO
- [x] clear specific state
- [x] update error alert design

### CODE REVIEW

**Comment by daiki44** [comment]
will use #157

File: src/store/withdraw.js [hunk 1]
```diff
@@ -31,16 +32,17 @@ export const withdrawReducer = createReducer(
...
       const client = await clientWrapper.getClient()
       if (!client) return
       await client.startWithdrawal(amountWei, tokenContractAddress)
       dispatch(setWithdrawProgress(WITHDRAW_PROGRESS.COMPLETE))
-      dispatch(getBalance())
+      dispatch(getL2Balance())
     } catch (error) {
-      console.log(error)
+      console.error(error)
+      dispatch(setWithdrawProgress(WITHDRAW_PROGRESS.ERROR))
```
Comment (tkmct) [inline src/store/withdraw.js:45]: why don't you save error object in reducer?

File: src/store/transfer.js [hunk 2]
```diff
@@ -52,37 +56,40 @@ export const transferReducer = createReducer(initialState, {
...
-    } catch (error) {
-      console.log(error)
-      dispatch(setTransferError(error.message))
-    } finally {
-      dispatch(setTransferIsSending(false))
+      dispatch(setTransferStatus(TRANSFER_PROGRESS.COMPLETE))
+      dispatch(pushToast({ message: 'Transfer success.', type: 'info' }))
+    } catch (e) {
+      console.error(e)
+      dispatch(setTransferStatus(TRANSFER_PROGRESS.ERROR))
```
Comment (tkmct) [inline src/store/transfer.js:74]: it's better to put error message using error obj.

**Comment by daiki44** [comment]
@tkmct Thanks! fixed! ac8b30e

**Comment by tkmct** [comment]
lgtm

**Pull request merged**